### ISSUE TITLE
Locon trained no error but no effect on generate

### ISSUE THREAD

**Issue by jirat-sumitanun** [bmaltais/kohya_ss#386]
All Extension that needed are installed and up to date.
I train on Google Colab, LoRA train is working perfectly fine but Locon have an issue.
even lower network_dim and conv_dim to train loha instead, give error too

for short details
pretrained_model SD 1.5 (chilloutmix)
resolution 512
max_train_type_value = 380
network_module=lycoris.kohya
unet_lr=1e-4
text_encoder_lr = 5e-5
network_dim=32
network_alpha=1
network_args "conv_dim=16" "conv_alpha=1" dropout=0.5 "algo=lora"

full code below
```
!accelerate launch \
  --config_file=$accelerate_config \
  --mixed_precision 'fp16' \
  --num_cpu_threads_per_process=2 \
  "train_network.py" \
  --cache_latents \
  --enable_bucket \
  --use_8bit_adam \
  $xformers \
  $mem_eff_attn \
  $shuffle_caption \
  $caption_extension \
  $keep_tokens \
  --pretrained_model_name_or_path=$model_path \
  --train_data_dir=$train_data_dir \
  --resolution=512,512 \
  --output_dir=$output_dir \
  --train_batch_size=$train_batch_size \
  --lr_scheduler=cosine \
  --lr_warmup_steps=20 \
  $max_train_type_value \
  --mixed_precision=fp16 \
  --save_every_n_epochs=$save_every_n_epochs \
  --seed=4242 \
  --save_precision=fp16 \
  --logging_dir=$logging_dir \
  --save_model_as=safetensors \
  --network_module=lycoris.kohya \
  --text_encoder_lr=5e-5 \
  --unet_lr=1e-4 \
  --network_dim=32 \
  --network_alpha=1 \
  --network_args "conv_dim=16" "conv_alpha=1" dropout=0.5 "algo=lora" \
  --clip_skip=2 \
  --output_name=$project_name
```

no issue when training
```
Use DreamBooth method.
prepare images.
found directory /content/drive/MyDrive/kohya_ss/datasets/shimoformi/20_shimoformi contains 19 image files
380 train images with repeating.
0 reg images.
no regularization images / 正則化画像が見つかりませんでした
[Dataset 0]
  batch_size: 5
  resolution: (512, 512)
  enable_bucket: True
  min_bucket_reso: 256
  max_bucket_reso: 1024
  bucket_reso_steps: 64
  bucket_no_upscale: False

  [Subset 0 of Dataset 0]
    image_dir: "/content/drive/MyDrive/kohya_ss/datasets/shimoformi/20_shimoformi"
    image_count: 19
    num_repeats: 20
    shuffle_caption: True
    keep_tokens: 1
    caption_dropout_rate: 0.0
    caption_dropout_every_n_epoches: 0
    caption_tag_dropout_rate: 0.0
    color_aug: False
    flip_aug: False
    face_crop_aug_range: None
    random_crop: False
    is_reg: False
    class_tokens: shimoformi
    caption_extension: .txt


[Dataset 0]
loading image sizes.
100% 19/19 [00:10<00:00,  1.85it/s]
make buckets
number of images (including repeats) / 各bucketの画像枚数（繰り返し回数を含む）
bucket 0: resolution (384, 640), count: 120
bucket 1: resolution (448, 576), count: 60
bucket 2: resolution (640, 384), count: 200
mean ar error (without repeats): 0.11718760636065231
prepare accelerator
Using accelerator 0.15.0 or above.
load StableDiffusion checkpoint
loading u-net: <All keys matched successfully>
loading vae: <All keys matched successfully>
Downloading (…)lve/main/config.json: 100% 4.52k/4.52k [00:00<00:00, 590kB/s]
Downloading pytorch_model.bin: 100% 1.71G/1.71G [00:12<00:00, 137MB/s]
loading text encoder: <All keys matched successfully>
Replace CrossAttention.forward to use FlashAttention (not xformers)
[Dataset 0]
caching latents.
100% 19/19 [00:12<00:00,  1.53it/s]
import network module: lycoris.kohya
Using rank adaptation algo: lora
Apply different lora dim for conv layer
Conv Dim: 16, Linear Dim: 32
Use Dropout value: 0.5
Create LyCORIS Module
create LyCORIS for Text Encoder: 72 modules.
Create LyCORIS Module
create LyCORIS for U-Net: 278 modules.
enable LyCORIS for text encoder
enable LyCORIS for U-Net
prepare optimizer, data loader etc.

use 8-bit AdamW optimizer | {}
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 380
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 76
  num epochs / epoch数: 5
  batch size per device / バッチサイズ: 5
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 380
steps:   0% 0/380 [00:00<?, ?it/s]epoch 1/5
steps:  20% 76/380 [03:05<12:23,  2.45s/it, loss=0.356]epoch 2/5
steps:  40% 152/380 [06:12<09:19,  2.45s/it, loss=0.259]saving checkpoint: /content/drive/MyDrive/kohya_ss/output/shimoformi/shimoformi-000002.safetensors
saving state.
epoch 3/5
steps:  60% 228/380 [09:45<06:30,  2.57s/it, loss=0.245]epoch 4/5
steps:  80% 304/380 [12:52<03:13,  2.54s/it, loss=0.25] saving checkpoint: /content/drive/MyDrive/kohya_ss/output/shimoformi/shimoformi-000004.safetensors
saving state.
epoch 5/5
steps: 100% 380/380 [16:22<00:00,  2.59s/it, loss=0.237]saving last state.
save trained model to /content/drive/MyDrive/kohya_ss/output/shimoformi/shimoformi.safetensors
model saved.
steps: 100% 380/380 [16:42<00:00,  2.64s/it, loss=0.237]
```
When generate with Locon that add with extra network style prompt `<lora:shomiformi:1>`, no error print out
When generate with Locon that add with sd-webui-additional-networks, it will show error as below.
The result is the same no effect on generation.
![Screenshot 2023-03-15 141314](https://user-images.githubusercontent.com/20882499/225234050-227a77e4-4819-4578-984f-7d096647b7a2.png)
![Screenshot 2023-03-15 141328](https://user-images.githubusercontent.com/20882499/225234083-968144bd-9c56-451d-b061-8a88ee4f1e09.png)

```
weight's shape is different: lora_unet_input_blocks_3_0_op.lora_down.weight expected torch.Size([16, 320, 3, 3]) found torch.Size([16, 320]). SD version may be different
weight's shape is different: lora_unet_input_blocks_1_0_in_layers_2.lora_down.weight expected torch.Size([16, 320, 3, 3]) found torch.Size([16, 320]). SD version may be different
weight's shape is different: lora_unet_input_blocks_1_0_out_layers_3.lora_down.weight expected torch.Size([16, 320, 3, 3]) found torch.Size([16, 320]). SD version may be different
weight's shape is different: lora_unet_input_blocks_2_0_in_layers_2.lora_down.weight expected torch.Size([16, 320, 3, 3]) found torch.Size([16, 320]). SD version may be different
...
shapes for 50 weights are converted.
LoRA model shimoformi(40a49adc7b83) loaded: _IncompatibleKeys(missing_keys=['lora_unet_input_blocks_1_0_in_layers_2.lora_down.weight', 'lora_unet_input_blocks_1_0_out_layers_3.lora_down.weight', 'lora_unet_input_blocks_2_0_in_layers_2.lora_down.weight', 'lora_unet_input_blocks_2_0_out_layers_3.lora_down.weight',...
```

I don't have error with other Locon that download from civitai, it's work fine
![Screenshot 2023-03-15 142656](https://user-images.githubusercontent.com/20882499/225236766-d4f9175f-b466-4163-a112-b26f6dc9dcbc.png)

**Comment by bmaltais** [comment]
Hello @jirat-sumitanun You should open this issue directly on kohya_ss sd-script repo as this is related to his trainer code and not the GUI that launch it: https://github.com/kohya-ss/sd-scripts/issues
